src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -794,8 +794,9 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 			Overwrite="true"
 	/>
 	<ItemGroup>
-		<_CompiledFlataArchive Include="$(IntermediateOutputPath)\**\*.flata"/>
-		<_CompiledFlataStamp Include="$(IntermediateOutputPath)\**\compiled.stamp"/>
+		<_CompiledFlataArchive Include="$(IntermediateOutputPath)\lp\**\*.flata" Condition="'$(_Aapt2Version)' != '@(_Aapt2VersionCache)'" />
+		<_CompiledFlataArchive Include="$(IntermediateOutputPath)\*.flata" Condition="'$(_Aapt2Version)' != '@(_Aapt2VersionCache)'" />
+		<_CompiledFlataStamp Include="$(IntermediateOutputPath)\lp\**\compiled.stamp" Condition="'$(_Aapt2Version)' != '@(_Aapt2VersionCache)'" />
```

Comment:
Maybe all of these should use `$(_AndroidLibrayProjectIntermediatePath)`? (aaah typo)

Then these would still work if someone has `$(UseShortFileNames)=False`.